### ISSUE TITLE
Footer visible on ms_message

### ISSUE THREAD

**Issue by aawdls** [dls-controls/machine-status#2]
![image](https://user-images.githubusercontent.com/17617270/155168135-e90faa0b-40d3-4b21-bfd7-88767d7bf389.png)

On ms_message screen, the footer appears and obscures part of the bottom of the screen.

**Comment by aawdls** [comment]
On a monitor of the resolution we'll use in production, it isn't so bad. I'm not sure if it's the footer component, or just the way the scaling / zoom is done.

**Comment by rjwills28** [comment]
I suspect it is the latter as the footer should not be included in this mode. This seems to be the only screen where the displayed width is greater than the height and so the amount of scaling applied is being dictated by the width (compared to the monitor size). However I would still not expect the 'blue band' to actually cover the screen so I will look into this.

**Comment by rjwills28** [comment]
@aawdls , I am having trouble reproducing this issue. I have tried changing the browser dimension to multiple ways but do not see the blue band at the bottom. Is this seen only on the machine status displays or do you see it on a local browser? Can I check what version of cs-web-lib this has been built with?

**Comment by rjwills28** [comment]
Just checked and it looks like it is built with cs-web-lib version 0.1.5 which should be correct and contain the scaling fix.

**Comment by aawdls** [comment]
It's not very noticeable on the status display monitor which is 1280x1024 (photo attached) 

![image](https://user-images.githubusercontent.com/17617270/155725521-2ca0ffd0-2f37-42ed-a6ac-49cbd2abc780.png)

On wider monitors it becomes more noticeable e.g. mine which is 1920x1200 (photo above) and @MJGaughran 's which is super wide (? Martin what's your screen resolution?)

Since we only need to use this scaling mode on the display version, would a possible workaround just be to ensure all the opi files have the same aspect ratio?

**Comment by rjwills28** [comment]
Thanks! I can see it on mine now when I enter full screen mode.

**Comment by rjwills28** [comment]
Indeed setting the height in the OPI would fix this issue but I did want to look into this in a bit more detail in case it comes up again. Essentially I think what was happening was the if the OPI width and/or height was less than the browser height/width then it was scaling the opi image up. This led the opi image to become bigger than the allocated height and so it got cut off. It is important that this does not happen therefore I have updated the cs-web-lib method so that it only scales the opi image down if it is larger than the display. See PR: https://github.com/dls-controls/cs-web-lib/pull/3. This fix means that you will not lose the bottom of the opi which is the most important issue here.

A separate issue related to this is that the background app colour was set to blue, hence you see the blue band at the bottom. For consistency I think it is better to set this colour to grey so that it matches the background of the OPI screens. Therefore you will not see a ribbon. This is purely a cosmetic issue and I think is applicable for the machine status screens. See PR: https://github.com/dls-controls/machine-status/pull/13
